composer.json

```diff
@@ -14,7 +14,7 @@
         }
     ],
     "require": {
-        "php": "^5.5.9 || ^7.0"
+        "php": "^5.6 || ^7.0"
```

Comment:
Please keep 5.5.9.

Reply:
Yeah, this was mainly just laziness on my part - I should add proper templated type inference for `call_user_func_array`

Reply:
As long as phpstan can work for callers to the public interface of the code, all is well. I don't care so much that phpstan has some trouble with the internals, so keeping `call_user_func_array` is fine for now.